core-feature-pack/common/pom.xml

```diff
@@ -301,10 +301,6 @@
             <groupId>org.wildfly.security</groupId>
             <artifactId>wildfly-elytron-credential</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.wildfly.security</groupId>
-            <artifactId>wildfly-elytron-credential-source-deprecated</artifactId>
-        </dependency>
```

Comment:
There are also a few more dependencies on other deprecated Elytron modules that should be removed here as well:
* `wildfly-elytron-auth-server-deprecated`
* `wildfly-elytron-http-deprecated`
* `wildfly-elytron-sasl-deprecated`
* `wildfly-elytron-x500-deprecated`

Reply:
The dependency on `wildfly-elytron-x500-deprecated` in `elytron/pom.xml` should also be removed.

Reply:
The `wildfly-elytron-auth-server-deprecated` dependency should also be removed from `protocol/pom.xml` and `remoting/subsystem/pom.xml`.